examples/cassandra/README.md

```diff
@@ -84,7 +84,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          image: gcr.io/google_containers/cassandra:v7
+          image: gcr.io/google_containers/cassandra:v8
```

Comment:
We'll need to do this as a separate PR to ensure that we don't break users who read these docs before the image is pushed...